core/blockly.ts

```diff
@@ -315,14 +315,19 @@ export function hideChaff(opt_onlyClosePopups?: boolean) {
  * Returns the main workspace.  Returns the last used main workspace (based on
  * focus).  Try not to use this function, particularly if there are multiple
  * Blockly instances on a page.
- * @return The main workspace.
  * @see Blockly.common.getMainWorkspace
  * @alias Blockly.getMainWorkspace
  */
 // AnyDuringMigration because:  Property 'getMainWorkspace' does not exist on
 // type 'void'.
 export const getMainWorkspace = (common as AnyDuringMigration).getMainWorkspace;
```

Comment:
Is this cast actually necessary or can it be removed?

Reply:
Removed all instances of AnyDuringMigration from `blockly.ts` except for the one in `populateProcedures` since I want to get rid of that as part of the shared procedures project anyway, and it's not external facing.